### DISCUSSION TITLE
The cloud server uses mitmweb, how to open the management web

### DISCUSSION THREAD

**Discussion by freshworker** [mitmproxy/mitmproxy#6142] [Q&A]
Excuse me，The cloud server uses mitmweb, how to open the management web (public IP address +8081 cannot open)?
**The cloud server does not have a browser**, so it cannot be opened directly as 127.0.0.1:8081.

 Can I modify the mitmweb configuration file and change the management web address to the public Ip address of the cloud server itself :8081 for the operation of mitmweb.

Thanks!

![微信截图_20230523163643](https://github.com/mitmproxy/mitmproxy/assets/77287677/accf6f8a-fc0c-4ca6-a3fa-c71d4c0528d4)

**Comment by mhils** [comment]
**First option:** You can modify the [`web_host`](https://docs.mitmproxy.org/stable/concepts-options/#web_host) option to change the IP address mitmproxy is listening on. However, be careful about exposing your mitmproxy instance to the public internet.

**Second option:** Use SSH to forward port 8081 to your local machine:
```
ssh -NL 8081:127.0.0.1:8081 user@your-server.example.com
```
and then connect to http://127.0.0.1:8081 on your local machine.

**Reply by freshworker** [reply]
Excuse me， about exposing  mitmproxy instance to the public internet，Whether to use  `--set block_global=false`  for configuration.

**Reply by freshworker** [reply]
Excuse me，About second option,my local machine doesn't have a public IP address.
